test/common/schemaform/definitions/dateRange.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { schema, uiSchema } from '../../../../src/js/common/schemaform/definitions/dateRange';
+
+describe('Schemaform definition dateRange', () => {
+  it('should render dateRange', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          uiSchema={uiSchema()}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('label').length).to.equal(8);
+    expect(formDOM.querySelectorAll('input').length).to.equal(2);
```

Comment:
Your call on the level of detail to test--I just noticed that the other tests all verify that all of the input elements are present.